code/modules/cargo/export_scanner.dm

```diff
@@ -1,22 +1,159 @@
+
 /obj/item/export_scanner
```

Comment:
could you change the path to reflect the object better for admins or mappers or whatever?